test/run-integration-tests

```diff
@@ -6,7 +6,7 @@ set -eo pipefail
 
 declare project_root="${0%/*}/.."
 declare ringpop_common_dir="${0%/*}/ringpop-common"
-declare ringpop_common_branch="master"
+declare ringpop_common_branch="feature/admin-reap"
```

Comment:
I completely agree. That branch needs to land to master in `ringpop-common` before we merge this branch.